### ISSUE TITLE
Entry is being hidden in Keyboard

### ISSUE THREAD

**Issue by prakharshrivastava2** [lucacivale/Maui.BottomSheet#22]
<img width="349" alt="Screenshot 2024-12-03 at 10 39 46 PM" src="https://github.com/user-attachments/assets/a0d78a10-ba14-4048-a114-0d6d65b098ef">

When I make Entry and try to write something, then it is being hidden in Keyboard.

**Comment by lucacivale** [comment]
@prakharshrivastava2 can you provide a small repro for me?
But this looks like the normal platform behavior. Seems like your entry isn't actually hidden, only part of the input container.

**Comment by prakharshrivastava2** [comment]
Below code is under a StackLayout in a ContentPage. You may see that other things are alos being hidden like Buttons ...
 
 
 <bottomsheet:BottomSheet x:Name="bottom" IsDraggable="True" SheetStates="Large" HasHandle="True">
                      
       <bottomsheet:BottomSheet.ContentTemplate>
        <DataTemplate>



            
            <StackLayout 
                         >
                <Label Text="LOGIN OR SIGNUP" HorizontalOptions="CenterAndExpand"  TextColor="Black" Margin="30" FontSize="18"  />

                  <Label Text="Unlock coupons, profile and much more" HorizontalOptions="StartAndExpand"  TextColor="Black" Margin="20,0,0,0" FontSize="14"  />


                  <inputLayout:SfTextInputLayout Hint="Mobile Number"
                                               Margin="20,10,20,0"
                               ContainerType="Outlined"
                                              
                                               ContainerBackground="White"
                               OutlineCornerRadius="0">
                     <Entry Text="{Binding Mobile_Number, Mode=TwoWay}" MaxLength="10" Keyboard="Numeric" HeightRequest="30"/>
                </inputLayout:SfTextInputLayout>

                <Button CornerRadius="0" Clicked="Login_Button_Clicked" Text="VERIFY" TextColor="White" BackgroundColor="Black" HorizontalOptions="FillAndExpand" VerticalOptions="FillAndExpand" HeightRequest="50" Margin="20"/>

                 <Label Text="By continuing, you agree to the terms of serice and privacy policy" HorizontalOptions="CenterAndExpand"  TextColor="Black" Margin="10,0,10,0" FontSize="10"  />

                 
            </StackLayout>

    
        </DataTemplate>
    </bottomsheet:BottomSheet.ContentTemplate>
</bottomsheet:BottomSheet>

**Comment by prakharshrivastava2** [comment]
<img width="346" alt="Screenshot 2024-12-22 at 4 16 12 AM" src="https://github.com/user-attachments/assets/0a8c1291-d637-44df-bdfc-f012f62ff951" />


I used The49.Maui.BottomSheet library all issues resolved, no keyboard overlapping. Even If I create multiple sheets and try to call them one by one , then on IOS all sheets does not call. This issues I did not faced in The49.Maui.BottomSheet. This libarary have multiple issues. I wasted a lot of time and reputation for my customer.
